js/enviro_phat.py

```diff
@@ -0,0 +1,38 @@
+''' Copyright 2017 Vprime
```

Comment:
This should go under `js/envirophat` too.

Reply:
Oh, it's there. Then you need to `git rm` this file.

Reply:
that's right. same thing here as well. give it some time and see if my change gets reflected.